fontforgeexe/charinfo.c

```diff
@@ -3573,31 +3573,39 @@ static int TeX_Default(GGadget *g, GEvent *e) {
 	style = MacStyleCode(sf,NULL);
 
 	if ( cid == CID_TeX_HeightD ) {
+	    mcid = CID_TeX_Height;
 	    if ( basesc!=ci->sc && basesc->tex_height!=TEX_UNDEF )
 		value = basesc->tex_height;
 	    else
 		value = rint(b.maxy);
 	    if ( value<0 ) value = 0;
 	} else if ( cid == CID_TeX_DepthD ) {
+	    mcid = CID_TeX_Depth;
 	    if ( basesc!=ci->sc && basesc->tex_depth!=TEX_UNDEF )
 		value = basesc->tex_depth;
 	    else {
 		value = -rint(b.miny);
 		if ( value<5 ) value = 0;
 	    }
 	} else if ( cid == CID_HorAccentD ) {
+	    mcid = CID_HorAccent;
 	    double italic_off = (b.maxy-b.miny)*tan(-sf->italicangle);
 	    if ( b.maxx-b.minx-italic_off < 0 )
 		value = rint(b.minx + (b.maxx-b.minx)/2);
 	    else
 		value = rint(b.minx + italic_off + (b.maxx - b.minx - italic_off)/2);
-	} else if ( (style&sf_italic) || sf->italicangle!=0 ) {
-	    value = rint((b.maxx-ci->sc->width) +
-			    (sf->ascent+sf->descent)/16.0);
-	} else
-	    value = 0;
-	sprintf( buf, "%d", value );
-	GGadgetSetTitle8(GWidgetGetControl(ci->gw,cid-5),buf);
```

Comment:
So it was effectively an off by one error here?

Reply:
Specifically -4, I think, but yes. 

I feel like one can be this clever if one gets it right (and it stays right), otherwise the explicit tokens go in.